cmd/serverless/main.go

```diff
@@ -247,6 +249,19 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		}
 	}()
 
+	// start appsec
```

Comment:
Do we want to extract that to a function, and gate that behind a feature flag (environment variable)?
Do this impact the coldstart? If yes, I guess some customers may not want to have this feature and should not be cold-start impacted?

Reply:
Oh I've just seen that it's checked a bit deeper in the code

Reply:
Yes, it's indeed hidden inside, because I wanted to keep the integration as simple as possible.
So right now you can see it is disabled when there's no error but the returned appsec pointer is nil.